### ISSUE TITLE
Add support for zoom

### ISSUE THREAD

**Issue by jschwarty** [nrwl/angular-console#93]
Looks like `command +` and `command -` do not work to zoom in and out in the app.

Also, it looks like the `width: 100vw;` on the `.nested-outlet-container` class may be an issue when zooming. The schematics options screen does not scale correctly when the css `zoom` property is used on a parent element like `<body>` (not sure if css `zoom` is the same as the zoom that the browser uses with `command +`, but if it is then this may be an issue).

Oh, and one more. When using the css `zoom` on the `<body>` it results in a vertical scroll bar. My guess is that this is due to the structure of the layout with the sticky header/sidebar and a scrollable panel. Just want to add that info here to make sure that gets tested when zoom is implemented. 😄

**Comment by mrmeku** [comment]
@jschwarty We will likely need to use this API to make zoom work the way it would in a browser. It should be pretty straightforward to implement if we just capture ctrl-/+ events and push those down to our electron server and hook into the API. If you (or anyone else) feel like giving this go that would be awesome

**Comment by mrmeku** [comment]
We have decided to sunset our electron release of console so that we can better focus on high quality editor extension. That being the case I'm going to sadly close this bug with no plans to resolve the ask.
